apps/ledger-live-mobile/src/screens/Accounts/index.tsx

```diff
@@ -83,6 +84,19 @@ function Accounts({ navigation, route }: Props) {
           }
         }
       }
+
+      if (params.id) {
+        const account = accounts.find(acc => acc.id === params.id);
+
+        if (account) {
+          // reset params so when we come back the redirection doesn't loop
+          navigation.setParams({ ...params, id: undefined });
+          navigation.navigate(ScreenName.Account, {
+            accountId: account.id,
+            isForwardedFromAccounts: true,
```

Comment:
what does this parameter do? I can't find any reference of it being used anywhere (this is a good example of why the navigation typing is needed)